src/kernels/kernelDependencyService.node.ts

```diff
@@ -247,12 +252,30 @@ export class KernelDependencyService implements IKernelDependencyService {
                 });
             }
             traceInfoIfCI(`Prompting user for install (this.isCodeSpace=${this.isCodeSpace}).`);
-            const selection = this.isCodeSpace
-                ? Common.install()
-                : await Promise.race([
-                      this.appShell.showInformationMessage(message, { modal: true }, ...options),
-                      promptCancellationPromise
-                  ]);
+            let selection;
+            do {
+                selection = this.isCodeSpace
+                    ? installOption
+                    : await Promise.race([
+                          this.appShell.showInformationMessage(message, { modal: true }, ...options),
+                          promptCancellationPromise
+                      ]);
+
+                if (selection === moreInfoOption) {
+                    sendTelemetryEvent(Telemetry.PythonModuleInstall, undefined, {
+                        action: 'moreInfo',
+                        moduleName: productNameForTelemetry,
+                        resourceType,
+                        resourceHash,
+                        pythonEnvType: interpreter.envType
+                    });
+
+                    // Link to our wiki page on jupyter kernels + ipykernel
+                    // https://github.com/microsoft/vscode-jupyter/wiki/Jupyter-Kernels-and-the-Jupyter-Extension#python-extension-and-ipykernel
+                    this.appShell.openUrl('https://aka.ms/AAhi594');
+                }
+                // "More Info" isn't a full valid response here, so reprompt after showing it
+            } while (selection === moreInfoOption);
```

Comment:
Loop here felt a bit funky, but I don't believe there is a way to keep open a modal dialog. Returning back a KernelDependencyResponse of "moreInfo" didn't seem right, so just ask again until we get a valid response (cancel, install, selectKernel).